package.json

```diff
@@ -7,15 +7,15 @@
   "scripts": {
     "bootstrap": "yarn",
     "bootstrap:ci": "yarn install --frozen-lockfile",
-    "build:all": "yarn build:crypto-dependencies && node ./scripts/turbo build",
+    "build:all": "node ./scripts/turbo build",
     "build:ci": "node ./scripts/turbo build",
-    "build:clients:generic": "node ./scripts/turbo build -F=@aws-sdk/aws-echo-service...",
-    "build:clients:since:release": "yarn build:packages && node ./scripts/turbo build $(lerna changed | grep -e '@aws-sdk/[client|lib]-*' | sed 's/^/ -F=/' | tr '\n' ' ')",
-    "build:crypto-dependencies": "node ./scripts/turbo build -F=@aws-sdk/types... -F=@aws-sdk/util-locate-window...",
-    "build:e2e": "yarn build:crypto-dependencies && node ./scripts/turbo build -F=@aws-sdk/client-cloudformation... -F=@aws-sdk/karma-credential-loader... -F=@aws-sdk/client-s3-control... -F=@aws-sdk/client-sts...",
-    "build:packages": "node ./scripts/turbo build -F=!@aws-sdk/client-*... -F=!@aws-sdk/aws-*... -F=!@aws-sdk/lib-*...",
-    "build:protocols": "yarn build:crypto-dependencies && node ./scripts/turbo build -F=@aws-sdk/aws-protocoltests-*...",
-    "build:server-protocols": "yarn build:crypto-dependencies && node ./scripts/turbo build -F=@aws-sdk/*-server...",
+    "build:clients:generic": "node ./scripts/turbo build -F=@aws-sdk/aws-echo-service",
+    "build:clients:since:release": "node ./scripts/turbo build -F=./packages/* $(lerna changed | grep -e '@aws-sdk/[client|lib]-*' | sed 's/^/ -F=/' | tr '\n' ' ')",
```

Comment:
minor issue: this seems to create 2 spaces between `-F` args

Reply:
I don't think `build:clients:since:release` is really necessary anymore.

It was created as a timesaving measure, but with turbo it's redundant. This lerna command also takes a bit of time to run.

Change this entry to simply `turbo -F=./clients/* -F=./lib/*`